src/js/components/Signature.jsx

```diff
@@ -5,14 +5,20 @@ import { ordinalHour } from '../utils/date.js'
 import Alert from './Alert.jsx'
 import { VALIDATED_WITH_ERRORS, SIGNED } from '../constants/statusCodes.js'
 
-const showReceipt = (code, timestamp, receipt) => {
+const showReceipt = ({status, timestamp, receipt, filingPeriod, email}) => {
+  const code = status.code
   if (code !== SIGNED) return null
 
   return (
-    <Alert type="success" heading="HMDA data accepted">
+    <Alert type="success" heading="HMDA filing completed">
       <div>
-        <p>Your submission was received on <strong>{ordinalHour(new Date(timestamp))}</strong>.</p>
-        <p>Your receipt number is <strong>{receipt}</strong>.</p>
+        <hr/>
+        <p>Congratulations, you have successfully completed your HMDA filing for {filingPeriod}!</p>
+        <p>Your data and signature have been received and recorded.</p>
+        <p className="usa-text-small">Your submission was received on <strong>{ordinalHour(new Date(timestamp))}</strong>.</p>
```

Comment:
Same as above here, what if we combine these to be `<p>Your submission was received on <strong>{ordinalHour(new Date(timestamp))}</strong> and Your receipt number is <strong>{receipt}</strong>.</p>`.